tests/jerry/es.next/math-cbrt.js

```diff
@@ -20,6 +20,11 @@ function isSameZero (x, y)
   return x === 0 && (1 / x) === (1 / y);
 }
 
+function approxEq (x, y)
+{
+  return Math.abs(x - y) < 1e-15
```

Comment:
How about using `Number.EPSILON` instead of a handcrafted one?

Reply:
Not sure, 1e-16 failed before.
Let's me have a try.
The EPSILON property has a value of approximately 2.2204460492503130808472633361816E-16

Reply:
[ 212/1145] FAIL (-6): tests/jerry/es.next/math-cbrt.js
================================================
Script Error: assertion failed

================================================

[summary] build/tests/jerry_tests-es.next-debug/local/bin/jerry tests/jerry

TOTAL: 1145
PASS: 1144
FAIL: 1

Success: 99%

Not working, how about Number.EPSILON * 10? or other number